src/main/java/Service.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class Service {
+    Scanner sc = new Scanner(System.in);
```

Comment:
🍏 Рекомендую не скупиться на буквы в названиях переменных, чтобы код был более читабельным
```suggestion
    Scanner scanner = new Scanner(System.in);
```